admin/client/src/components/Form/Form.js

```diff
@@ -1,27 +1,53 @@
-import React from 'react';
+import React, { PropTypes } from 'react';
 import SilverStripeComponent from 'lib/SilverStripeComponent';
+import MessageBox from 'components/MessageBox/MessageBox';
 
 class Form extends SilverStripeComponent {
 
-  constructor(props) {
-    super(props);
-
-    this.handleSubmit = this.handleSubmit.bind(this);
+  /**
+   * Generates a list of messages if any are available
+   *
+   * @returns {Array|null}
+   */
+  getMessages() {
```

Comment:
We should prefix methods returning React components with `render*()`. This is implied by the AirBnB conventions as well: https://github.com/airbnb/javascript/tree/master/react#ordering. So `renderMessages()` in this case.